protobuf/.gitattributes

```diff
@@ -0,0 +1,6 @@
+#
```

Comment:
this file should be present only in root module

Reply:
ok